app/views/favorites/index.html.erb

```diff
@@ -1,11 +1,5 @@
 <h1>Favorite Pets</h1>
 
-  <nav class="navigation">
```

Comment:
Nice work moving these to the application view it makes everything look much cleaner